SparkPost/2web2ui
# FAD-6414: Sanitize HTTP error messages

_Refer to [FAD-6414](https://jira.int.messagesystems.com/browse/FAD-6414)_

**Testing**

1. Navigate to change plan page
1. Select any paid plan
1. Fill-in all billing info fields as normal except for credit card enter an invalid card number, "123"
1. Click "Upgrade Plan" button

You should get an error message.

Was:
> Transaction declined.<br>CREDIT_CARD_NUMBER_H - Credit card type is not accepted by this merchant account.\nCredit card number must be 12-19 digits.

Now: 
> Transaction declined. CREDIT_CARD_NUMBER_H - Credit card type is not accepted by this merchant account. Credit card number must be 12-19 digits.

**Note:**  I looked at a number of solutions and packages.  The `createElement` solution was cool, but I feel it might leave us exposed to edge cases.  The node-html-to-text, striptags, and string-strip-html packages would work, but for different reasons are too much for what we need.
